packages/next/src/server/web-server.ts

```diff
@@ -289,6 +269,9 @@ export default class NextWebServer extends BaseServer<WebServerOptions> {
     }
 
     res.send()
```

Comment:
This is using a `WebNextResponse`:

https://github.com/vercel/next.js/blob/39fc06a622495d0a8a8ef9e037a68a5e4141fc37/packages/next/src/server/base-http/web.ts#L92-L95

So really all it's doing here is that when the `send()` is called is it's unblocking the reader so that the `pipeTo` operation can start pumping.